website/docs/registries/overview.md

```diff
@@ -0,0 +1,7 @@
+# Overview
+
+Scarb is not only a Cairo build toolchain but also a package manager, so it needs support for package storage. Currently, Git is the default for hosting community packages, but their visibility and ease of discovery are greatly limited. That's what registries are for, to have a concentration of packages in one place that are easily discoverable.
+
+Although the official Cairo package registry is in active development and not ready for use yet, you can [host your own](/docs/registries/custom-registry) registry or search for and [use](/docs/registries/custom-registry#using-custom-registry) a community-hosted one.
+
+If you are interested in learning about the packaging and publishing process, you can read about it [here](/docs/registries/packaging).
```

Comment:
Use [internal links](https://vitepress.dev/guide/markdown#internal-links). Editors will try to maintain these when moving files etc.